comms/src/protocol/rpc/status.rs

```diff
@@ -157,8 +171,10 @@ pub enum RpcStatusCode {
     MalformedResponse = 5,
     /// Misc. errors
     General = 6,
+    /// Entity not found
+    NotFound = 7,
     /// Unrecognised RPC status code
-    InvalidRpcStatusCode = 7,
+    InvalidRpcStatusCode = 100,
```

Comment:
```suggestion
    InvalidRpcStatusCode = 500,
```

Reply:
So this variant actually isn't a specific code, it stands for "anything else other than the above". The only reason it's there at all is because we have to put a number there. I should add a comment for that.

That is why moving this number to > 7 doesnt actually break anything (other than older clients will say invalid status code instead of not found). So we can make this whatever we want really, it's more what makes the code clearer. I would prefer it to be within a u8 though (it doesn't matter at all, but if in the future it does, it's better to save the byte)

Reply:
I think it'll actually be clearer if we leave the number off completely. Rust will auto assign it, and it's clearer that the variant doesn't represent a single code.